src/clj_kondo/impl/analyzer.clj

```diff
@@ -404,6 +426,15 @@
                                         :macro? macro?))
                              %)
                            bodies)
+        _ (when (some-> config :linters :bang-suffix-consistency :level #{:error :warning}) 
```

Comment:
There is a function for that:

https://github.com/borkdude/clj-kondo/pull/747/files#diff-a0f7c1f7683ffc1de22015480c7ce38dR271

I didn't include it my PR for your PR, but I think it would be good to add it after you merge it.

Reply:
I now included it in my PR as well.